resources/js/views/settings/users/View.vue

```diff
@@ -332,12 +361,25 @@ export default {
   mounted () {
     EventBus.$on('currentUserChangedEvent', this.togglePermissionFlags);
 
-    if (this.config.type !== 'profile') {
+    if (!['profile', 'view'].includes(this.config.type)) {
```

Comment:
you also don't need to load the roles if the user is editing himself (not via the profile but via the user page)